vocabularies/geological-sites.ttl

```diff
@@ -138,32 +143,69 @@ gsqst:colluvial a skos:Concept ;
 
 gsqst:outcrop a skos:Concept ;
     skos:broader gsqst:field-site ;
-    dcterms:source "Merriam-Webster Dictionary" ;
+    dct:source "Merriam-Webster Dictionary" ;
```

Comment:
their def isnt great so i'd remove this along with the new def.

Reply:
dictionary source removed